jax/lax/__init__.py

```diff
@@ -19,6 +19,7 @@
   DotDimensionNumbers,
   GatherDimensionNumbers,
   Precision,
+  RandomAlgorithm,
```

Comment:
My main problem is what to do here? I don't think I can (reasonably) condition it on `jax.lib.version >= (...)` because  importing `jax` in this `__init__` file feels pretty wrong.

One solution I can imagine is to define a default version for these functions in `lax.py` what just throws an error so they exists (so importing them will work fine) but don't do anything in runtime. What do you think?

Reply:
I think it's fine to just set `RandomAlgorithm` to
```
class RandomAlgorithm: pass
```
if the version is too old.

(This is only a temporary state, we can bump the minimum, but I'd get this PR in first.)